doc/dev/developer_guide/essentials.rst

```diff
@@ -21,7 +21,7 @@ RADOS     Neha Ojha        neha-ojha
 RGW       Yehuda Sadeh     yehudasa
 RGW       Matt Benjamin    mattbenjamin
 RBD       Ilya Dryomov     dis 
-CephFS    Patrick Donnelly batrick
+CephFS    Venky Shenkar    vshankar
```

Comment:
typo - should be Shankar

Reply:
Good catch, Josh. That'll show me to make changes at 1 a.m.